packages/tables/docs/02-getting-started.md

```diff
@@ -308,6 +308,77 @@ In this example, clicking on each post will take you to the `posts.edit` route.
 
 If you'd like to [override the URL](columns#opening-urls) for a specific column, or instead [run a Livewire action](columns#running-actions) when a column is clicked, see the [columns documentation](columns#opening-urls).
 
+## Row appearance
+
+To change the appearance of rows conditionally on the record data, you may add the `getTableRecordClassesUsing()` method.
+
+The `getTableRecordClassesUsing()` method may return a string or an array of classes that need to be applied to the row.
+
+```php
+<?php
```

Comment:
Let's keep this example simple and have it contain just the method (like the record URL example above).